build/testing/integration/authz/auth.go

```diff
@@ -0,0 +1,524 @@
+package authz
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.flipt.io/flipt/build/testing/integration"
+	"go.flipt.io/flipt/rpc/flipt"
+	"go.flipt.io/flipt/rpc/flipt/auth"
+	sdk "go.flipt.io/flipt/sdk/go"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+func Common(t *testing.T, opts integration.TestOpts) {
+	client := opts.TokenClient(t)
+
+	t.Run("Authentication Methods", func(t *testing.T) {
+		ctx := context.Background()
+
+		t.Run("List methods", func(t *testing.T) {
+			t.Log(`List methods (ensure at-least 1).`)
+
+			methods, err := client.Auth().PublicAuthenticationService().ListAuthenticationMethods(ctx)
+
+			require.NoError(t, err)
+
+			assert.NotEmpty(t, methods)
+		})
+
+		t.Run("Get Self", func(t *testing.T) {
+			authn, err := client.Auth().AuthenticationService().GetAuthenticationSelf(ctx)
+
+			require.NoError(t, err)
+
+			assert.NotEmpty(t, authn.Id)
+		})
+
+		for _, namespace := range integration.Namespaces {
+			t.Run(fmt.Sprintf("InNamespace(%q)", namespace.Key), func(t *testing.T) {
```

Comment:
```suggestion
			t.Run(fmt.Sprintf("InNamespace(%q)", namespace.Expected), func(t *testing.T) {
```

Will prevent `""` from showing in output

Reply:
This is to test both `default` and `""` (blank) explicitly. 
Otherwise, those two test suites will be deplicately named.